### ISSUE TITLE
Generator: use new error scheme

### ISSUE THREAD

**Issue by ReinvdZee** [victronenergy/gui-v2#1268]
To support the new error scheme, update the gui to use `/Error/n/Id` instead of `/ErrorCode`. See related gui-v1 [commit](https://github.com/victronenergy/gui/commit/65e28c113e716603df488e177e1a4b45c0f2d8aa)

**Comment by blammit** [comment]
Blocked until available in Venus OS.

**Comment by ReinvdZee** [comment]
Unblocked as the new error scheme implementation is merged per Venus OS v3.50~2

**Comment by ReinvdZee** [comment]
We also need to implement the new [error scheme](https://github.com/victronenergy/venus/wiki/dbus#error-codes), which is a bit more than just renaming the path. 

With the new error scheme, the gui also needs to parse the error. E.g., a third-party device (genset in this case) reports something like `dse:e-4098` , which means error code 4098 from manufacturer DSE.
The mapping is in veutil, and the gui needs to use that. You probably need something similar to [this](https://github.com/victronenergy/gui/commit/9030a2d9ba686d1e5eb57a89a9b39ef7e5ba043d)

**Comment by ReinvdZee** [comment]
I blocked this for now because we first need to work out the last specifics of the new error scheme.

**Comment by mpvader** [comment]
How does this work now @blammit & @chriadam ?

Pls, to unlock this for releasing gui-v2: remove any fields that show genset errors, also for dc gensets, in the detailed pages of the genset. Users can see errors in the notifications; which is a path that already works now.

@ReinvdZee fyi: I want to not now start implementing parsing error codes into gui's again, the whole plan was to centralize that into Venus OS.

Later, if needed, we can make something for this; but for now, my above unlock idea is good enough wrt usability for customers.

cc @martinbosma & @jhofstee

**Comment by mpvader** [comment]
ok decided on a slightly different plant: gui-v2 will look at all 8 paths, will show all the codes, and make it clickable and take user to notifications page where he can see the text explaining the code.
